packages/app_ui/lib/app_ui.dart

```diff
@@ -1,2 +1,5 @@
 library;
-export 'src/dialogs/dialogs.dart';
\ No newline at end of file
+export 'src/dialogs/dialogs.dart';
+export 'src/sizes/app_sizes.dart';
+export 'src/theme/app_theme.dart';
```

Comment:
use barrel here as well

Reply:
@kamilgadek you did not change anything here!
Please use barrels like that:

`export 'src/sizes/sizes.dart';`
`export 'src/theme/theme.dart';`